src-electron/db/query-command.js

```diff
@@ -1385,24 +1387,26 @@ async function selectAllCommandDetailsFromEnabledClusters(
       db,
       `
   SELECT
-    COMMAND.COMMAND_ID,
```

Comment:
Could we not re-arrange these? Will be easier to look at the changes and would be easier to rebase as well.

Reply:
I actually re-arranged them to make the entries more organized / readable. I can restore to previous state to avoid rebase issues.